### ISSUE TITLE
stack upgrade doesn't upgrade to the release mentioned in the bug report template

### ISSUE THREAD

**Issue by wereHamster** [commercialhaskell/stack#4326]
```
$ stack upgrade
Current Stack version: 1.7.1, available download version: 1.7.1
Skipping binary upgrade, you are already running the most recent version
```

This contradicts the bug report template which mentions 1.9.1 as latest release.

```
Please follow the steps below for reporting a bug:

Make sure that you are using the latest release (currently stack-1.9.1).
See the [upgrade instructions](http://docs.haskellstack.org/en/stable/install_and_upgrade/#upgrade) to upgrade.
```

**Comment by qrilka** [comment]
Looks to be duplicate of #4323

**Comment by mihaimaruseac** [comment]
Stack 1.9.1 is in release candidate phase at the moment. If you want it, you should `stack upgrade --git` or download it from the [releases page](https://github.com/commercialhaskell/stack/releases).

The release is now at 1.9.0.1 as it is only a candidate but when it will be fully released it will be named 1.9.1.

**Comment by mihaimaruseac** [comment]
Most likely. We were planning to get the release candidate fast but it got delayed.
